src/components/Page/Page.tsx

```diff
@@ -50,7 +56,7 @@ export class Page extends React.PureComponent<ComposedProps, never> {
   }
 
   componentDidUpdate(prevProps: ComposedProps) {
-    if (this.props.polaris.appBridge == null || this.titlebar == null) {
+    if (this.titlebar == null || this.delegateToAppbridge === false) {
```

Comment:
Would `!this.titlebar` work better here, since it also accounts for `this.titlebar` being `undefined`?  Is there a reason `undefined` should be treated differently from `null`, that I've glossed over?

Reply:
So, you're exactly right that `undefined` and `null` should be treated the same here. `==` actually does that for us.

Reply:
You're exactly right that `undefined` and `null` should be treated the same here, and in fact the reason to use `==` is so that we can check whether a reference is either `null` _or_ `undefined`.

Reply:
What you have is good.